native_client/java/jni/deepspeech.i

```diff
@@ -45,6 +45,7 @@
 %newobject DS_SpeechToText;
 %newobject DS_IntermediateDecode;
 %newobject DS_FinishStream;
+%newobject DS_ErrorCodeToErrorMessage;
```

Comment:
I don't see any use of that in the Java code ?

Reply:
Should I remove it?  Or I have to add it in other file @lissyx

Reply:
No, it's more that we should use it in java code. Maybe we don't properly check for errors right now, but that might be a good idea to start doing so.

Reply:
no idea yet